src/pages/library/DataBrowserPreview.js

```diff
@@ -52,7 +52,20 @@ const DataBrowserPreview = ({ id }) => {
   const [viewJSON, setViewJSON] = useState()
 
   const tableMap = _.keyBy('name', tables)
-  const tableNames = _.map('name', tables)
+  const selectOptions = _.flow([
+    _.partition(table => table.rowCount > 0),
+    _.toPairs,
+    _.flatMap(([rowIndex, tables]) => {
+      const sortedTables = _.flow([
+        _.sortBy('name'),
+        _.map(({ name, rowCount }) => { return { value: name, rowCount } })
+      ])(tables)
+
+      return rowIndex === '0' ?
+        sortedTables :
+        { label: 'Tables without data', options: sortedTables }
+    })
+  ])(tables)
```

Comment:
No arrays for flows plzz
```suggestion
  const selectOptions = _.flow(
    _.partition(table => table.rowCount > 0),
    _.toPairs,
    _.flatMap(([rowIndex, tables]) => {
      const sortedTables = _.flow(
        _.sortBy('name'),
        _.map(({ name, rowCount }) => { return { value: name, rowCount } })
      )(tables)

      return rowIndex === '0' ?
        sortedTables :
        { label: 'Tables without data', options: sortedTables }
    })
  )(tables)
```